### PR TITLE
Fs support

### PR DESCRIPTION
**Pull request by AnguIar**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -21,7 +25,12 @@ def __init__(self):
     def _start_service(self):
         self.log.info('Service is listening to broker: {0}, topic: {1}'.format(self.__config["kafka"]["host_ip"], self.__config["kafka"]["topic"]))
         try:
-            self.__task_handler.create_vrt_outputs_folder()
+            self.__utilities.create_folder(worker_constants.VRT_OUTPUT_FOLDER_NAME)
+            self.__utilities.create_folder(worker_constants.TILES_OUTPUT_FOLDER_NAME)
```
Comment (asafMasa) [inline src/main.py:29]: if FS

**Review by asafMasa — APPROVED** [review]

File: src/main.py [hunk 2]
```diff
@@ -24,11 +24,14 @@ def __init__(self):
...
+        storage_provider = self.__config['storage_provider'].upper()
         try:
             self.__utilities.create_folder(worker_constants.VRT_OUTPUT_FOLDER_NAME)
-            self.__utilities.create_folder(worker_constants.TILES_OUTPUT_FOLDER_NAME)
 
-            if (self.__config['storage_provider'].upper() == StorageProvider.S3):
+            if (storage_provider == StorageProvider.FS):
+                self.__utilities.create_folder(worker_constants.TILES_OUTPUT_FOLDER_NAME)
+
+            if (storage_provider == StorageProvider.S3):
```
Comment (asafMasa) [inline src/main.py:34]: why not to use "else"?

File: confd/production.tmpl [hunk 3]
```diff
@@ -27,7 +27,7 @@
     },
     "fs": {
         "external_physical_path": "{{ getv "/configuration/discrete/worker/fs/external/physical/path" "/ubuntu/outputs"}}",
-        "internal_outputs_path": "{{ getv "/configuration/discrete/worker/fs/internal/outputs/path" "/app/outputs"}}"
+        "internal_outputs_path": "{{ getv "/configuration/discrete/worker/fs/internal/outputs/path" "./tiles_outputs"}}"
```
Comment (asafMasa) [inline confd/production.tmpl:30]: change in toml file?

**Review by asafMasa — CHANGES_REQUESTED** [review]
🚀 Good job!

File: confd/production.tmpl (hunk 3 above)
Comment (AnguIar) [reply confd/production.tmpl:30]: we change the value, not the environment variable key
no need to change in toml file

**Review by asafMasa — APPROVED** [review]
👑

**Pull request merged**